charts/aad-pod-identity/templates/nmi-clusterrole.yaml

```diff
@@ -15,16 +15,16 @@ rules:
   verbs: ["get", "list"]
 - apiGroups: [""]
   resources: ["pods"]
-  verbs: ["get", "list"]
+  verbs: ["get", "list", "watch"]
```

Comment:
MIC role updates ?

Reply:
No role updates for MIC. MIC already had the required roles.